### ISSUE TITLE
Release new version

### ISSUE THREAD

**Issue by ZioberMichal** [super-csv/super-csv#115]
@kbilsted, @ash211 - next release is coming. I think that next version could be "2.5.0". What do you think?

**Comment by kbilsted** [comment]
we could start using semantic versioning :) but 2.5.0 is fine by me

**Comment by ash211** [comment]
Agreed, 2.5.0 feels like a better fit than the alternatives of a minor
point release (2.4.1) or a major release (3.0.0)

**Comment by kbilsted** [comment]
If there are breaking changes i like it to be 3.0

**Comment by ZioberMichal** [comment]
Thanks for the conformation. Next version is 2.5.0. Please give me few days I have to remind how I release previous version.

**Comment by kbilsted** [comment]
@ZioberMichal  any news?

**Comment by ZioberMichal** [comment]
Sorry, in meantime when I was not in project I had changed computer and had to set up new dev environment but I am really close to finish. Today I will try to generate new version but first I have to set up PGP app. I will be informing. Sorry for delay!

**Comment by kbilsted** [comment]
@ZioberMichal  no problem. We all have lives outside this project ;) is there a step-by-step description of what needs be set up in the repo in order to be a release manager? Last  I looked I didn't find it..

**Comment by ZioberMichal** [comment]
We have something like this: https://github.com/super-csv/super-csv/wiki/Performing-a-release. I have set up new dev environment but now I have to remind how to work with GIT on GitHub :) (In day to day work outside open source projects I use Mercurial)

**Comment by kbilsted** [comment]
Use gitextensions it's a cool app :-p

**Comment by thomas-mc-work** [comment]
Is there anything that is preventing the new release?

**Comment by ZioberMichal** [comment]
Since @dota17 updated release notes in task #184 I think we are ready to do that.

**Comment by patric-r** [comment]
I would love to finally see a new release of SuperCSV...
